sr/comp/cli/show_league_table.py

```diff
@@ -0,0 +1,55 @@
+__description__ = "Show the current state of the league table"
+
+
+SORT_TYPES = {  # cli string: (sort column, sort direction)
+    'rank': (0, False),
+    'game': (2, True),
+    'team': (3, False),
+}
+
+def command(settings):
+    import os.path
+
+    from sr.comp.comp import SRComp
+
+    comp = SRComp(os.path.realpath(settings.compstate))
+
+    league_table = []
+    for team in comp.teams.values():
+        scores = comp.scores.league.teams[team.tla]
+        league_pos = comp.scores.league.positions[team.tla]
+        league_table.append((
+            league_pos,
+            scores.league_points,
+            scores.game_points,
+            team.tla,
+            team.name,
+        ))
+
+    # sort rows
+    sort_col, sort_direction = SORT_TYPES[settings.sort]
+    league_table.sort(key=lambda row: row[sort_col], reverse=sort_direction)
+
+    # Print header
+    print("Rank | League | Game | Team")
+    for row in league_table:
+        print(f"{row[0]:>4} | {row[1]:>6} | {row[2]:>4} | {row[3]+':':<5} {row[4]}")
```

Comment:
Small suggestion -- would be great if we could indicate tied positions. Typically we annotate that as `=2` for ties for second place.

Reply:
I've added this in 45b9c97.